content/pages/education/apply-for-education-benefits/application/1990.md

```diff
@@ -5,12 +5,15 @@ layout: page-react.html
 description: Use your VA education benefits to pay for college or training programs. Find out which documents you’ll need to apply for benefits, and start your online application today.
 ---
 <div id="main">
-  <nav class="va-nav-breadcrumbs">
-    <ul class="row va-nav-breadcrumbs-list columns" role="menubar" aria-label="Primary">
+  <nav aria-label="Breadcrumb" aria-live="polite" aria-relevant="additions text" class="va-nav-breadcrumbs js-visual"
+  id="va-breadcrumbs">
+    <p class="usa-sr-only">Breadcrumb navigation will usually show all page links. It will adjust to show only the previous page when zoomed in, or viewed on a mobile device.</p>
+    <ol class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
       <li><a href="/">Home</a></li>
       <li><a href="/education/">Education</a></li>
-      <li class="parent"><a href="/education/apply-for-education-benefits/">Apply for Education Benefits</a></li>
-    </ul>
+      <li><a href="/education/apply/">Application Process</a></li>
+      <li><a aria-current="page"  href="/education/apply-for-education-benefits/application/1990/introduction">Apply for Education Benefits</a></li>
```

Comment:
As you go through the form, this link is going to be inaccurate, since the introduction page is just the first page of the form. I think our typical pattern in the site is for breadcrumbs to not include a link to the current page. Are we changing that? If we are, we may need to move the breadcrumbs into React component for all the React apps, so we can keep them up to date.

Reply:
@jbalboni Good call. The final breadcrumb link is being put back, but we're using CSS to remove `pointer-events`, and no, the breadcrumb should not change as you move through multi-step forms. I'll update this href to `/education/apply-for-education-benefits/application/1990`